sdk/core/Azure.Core.TestFramework/src/TestEnvironment.cs

```diff
@@ -42,25 +50,31 @@ protected TestEnvironment()
 
             var testProject = GetSourcePath(GetType().Assembly);
             var sdkDirectory = Path.GetFullPath(Path.Combine(RepositoryRoot, "sdk"));
-            var serviceName = Path.GetFullPath(testProject)
+            _serviceName = Path.GetFullPath(testProject)
```

Comment:
If we are making this variable static we should move the assignment into a static ctor.

Reply:
And possibly cache all other data that is loaded in this ctor.

Reply:
It's not static.... my bad.